package.json

```diff
@@ -146,13 +146,13 @@
     "ts-loader": "^9.2.6",
     "ts-node": "^10.1.0",
     "tsup": "^5.12.8",
-    "typescript": "^4.8.2",
+    "typescript": "^4.9.3",
     "url-loader": "^4.1.1",
     "webpack": "^5.68.0",
     "webpack-dev-server": "^3.11.2"
   },
   "peerDependencies": {
-    "typescript": ">= 4.2.x <= 4.8.x"
+    "typescript": ">= 4.4.x <= 4.9.x"
```

Comment:
Do you happen to know how old TS 4.3 is? I wonder if it's safe to drop support for it.

Reply:
TS 4.2 is 1 year and 9 months old, TS 4.3 is 1 year and 6 months old. According to the release notes, no breaking changes would block migration to TS 4.4.
See [TS 4.3 breaking changes](https://devblogs.microsoft.com/typescript/announcing-typescript-4-3/#breaking-changes) and [TS 4.4 breaking changes](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#breaking-changes).

Reply:
Thanks for looking that up! That's lovely, let's drop their support and introduce 4.9.